### ISSUE TITLE
Separate-uncertainty changes magnitude of uncertainty value

### ISSUE THREAD

**Issue by docingstation** [josephwright/siunitx#465]
I found an issue with the separate-uncertainty switch with siunix v3.0.7. Setting this to 'true' actually changes the uncertainty value!

MWE:
```latex
\documentclass[]{article}
\usepackage{siunitx}
\sisetup{
	separate-uncertainty,
}
\begin{document}
\qty{2600+-50}{\newton}

\qty[separate-uncertainty=false]{2600+-50}{\newton}
\end{document}
```

The first statement yields (2600 +- 5000) N while the second one yields the correct 2600(50) N.

**Comment by josephwright** [comment]
Hmm, this looks tricky - I will need to think a little about it.

**Comment by josephwright** [comment]
I need to check if this shows in v2 and if not why not (and what happens differently). Basically the issue is that with an uncertainty purely in the integer part, it's tricky doing the 'reconstruction' where the significant digits are `0`.

**Comment by docingstation** [comment]
v2 does not show this behavior

**Comment by josephwright** [comment]
@docingstation Sure, but I know that there was a reason I had to make an adjustment, and they use the same internal format for uncertainties ... so something is odd. May not get to this today: I have a few other urgent-ish tasks.

**Comment by docingstation** [comment]
Sure, no worries. I worked around it for the time being. Just wanted to let you know.

**Comment by josephwright** [comment]
Right, the issue I was thinking that relates is https://github.com/josephwright/siunitx/issues/454 - let me see what's going on.

**Comment by josephwright** [comment]
OK, I see the issue - it comes down to how one interprets `12300(10)` or similar. I think on balance I need to stick to that being equal to `12300 \pm 10` rather than `12300 \pm 1000`. That means I need to re-fix #454, which I'll do be re-opening it and closing here.
